redis_ratelimit/decorators.py

```diff
@@ -42,7 +42,7 @@ def is_rate_limited(request, rate=None):
             return True
 
     value = r.incr(redis_key)
-    if value == 1:
+    if value >= 1 and r.ttl(redis_key) is None:
```

Comment:
I think this can just be ` if r.ttl(redis_key) is None:` - the value should never be 0 straight after an `incr`

Reply:
You found the issue, nice! As David said, we can assume that it's never 0.